### DISCUSSION TITLE
Wood Hut Remastered

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#327] [Map Releases]
## This is a manual migration of forum user true nightmare's map release `WoodHut_Remastered`. Below contains the original post content. You can view the original post [here](https://nzportable.forumotion.com/t1982-land-v1-2-release)

Kept you waiting?

![0](https://github.com/nzp-team/nzportable/assets/26030250/4bb19d52-f7c2-456b-8016-a3f8f0f5f365)

10 months ago I got asked by MotoLegacy to make new mapping tutorials using trench broom, this didn't and won't be happening. But it did make me look into trench boom, after getting to grips with it I decided to recreate wood hut for the 1 million times using trench boom.

and it's finally ready.

![1](https://github.com/nzp-team/nzportable/assets/26030250/5dafe383-8836-4adb-859f-67e1a613b5e3)

Created from the ground up, this version of wood hut is a complete recreation of the map, and I would like to think it would make everyone who played the original happy. this is how I remember it looking when it was first created anyway.

![2](https://github.com/nzp-team/nzportable/assets/26030250/b0f0aeb6-b01d-4930-aa56-9212a176326d)

I look forward to seeing what you all think. I do not plan on making more maps nor creating tutorials for NZP. this game has always had a special place in my heart but I just don't have the time for it anymore.

Thank you all.

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/12148064/WoodHut_Remastered.zip)

**Comment by owenjames2023** [comment]
I’m testing it on PSP for ya

**Comment by owenjames2023** [comment]
Works fine has no problems on a psp 3000
